### ISSUE TITLE
Support for auto_now?

### ISSUE THREAD

**Issue by bykof** [aykut/django-bulk-update#25]
Hey,
does django-bulk-update support DateTimeFields with auto_now?
I tested it and django-bulk-update doesn't update the "updated" Field in my model which is a models.DateTimeField(auto_now=True) field.
Do you know how to fix this issue?

**Comment by aykut** [comment]
Hi,
https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.DateField.auto_now
Django respects `auto_now` option only when `save` is called like pre-save and post-save signals. It doesn't automatically update the field when you call `update`. Because `update` is considered a low-level SQL thing and it passes the exact values into db directly without any validations. So same thing actually goes for `bulk-update`, it would be overkill to check all fields for `auto_now` or `default` options.

In short, you can set date values explicitly before you call `bulk-update`.
Ex:
```
now = datetime.datetime.now()
for obj in objs:
    obj.datetime_field = now
```

**Comment by bykof** [comment]
Okay, this is exactly what I thought too :) 
Thank you very much for your support and keep going developing on django-bulk-update it's great! :+1:

**Comment by aykut** [comment]
Thanks :).
